src/clients/web3/useMulticall.ts

```diff
@@ -1,11 +1,22 @@
 import { Multicall } from 'ethereum-multicall';
 import { useMemo } from 'react';
 
-import { useWeb3 } from 'clients/web3';
+import { useAuth } from 'context/AuthContext';
 
 const useMulticall = () => {
-  const web3 = useWeb3();
-  return useMemo(() => new Multicall({ web3Instance: web3, tryAggregate: true }), [web3]);
+  const { provider } = useAuth();
+  return useMemo(
+    () =>
+      new Multicall({
+        // @ts-expect-error The Provider type used by Multicall only has one
```

Comment:
It would make more sense to add that property to the provider so that you don't unintentionally suppress other errors

Reply:
Not in this situation as it could lead to more errors if we ever use the modified version to send transactions

Reply:
ahh I see I misunderstood I thought this was an extra property that multicall needed. You can still solve this by upgrading ethers to match the version (5.7.2) used by multicall

Reply:
Good point! I'll make the update